src/components/Navbar.jsx

```diff
@@ -1,96 +1,104 @@
-import React, { useState } from 'react';
-import { FaBars, FaTimes } from 'react-icons/fa';
+import React, { useState } from "react";
+import { FaBars, FaTimes } from "react-icons/fa";
 
 const navlinks = [
-    {
-        title: "Download",
-        link: "/#download",
-    },
-    {
-        title: "Documentation",
-        link: "https://opensource.tcetmumbai.in/docs/projects/tcet-linux/about-tcet-linux",
-        newtab: true
-    },
-    {
-        title: "About Us",
-        link: "https://opensource.tcetmumbai.in/docs/about-tcetopensource",
-        newtab: true
-    },
+  {
+    title: "Download",
+    link: "/#download",
+  },
+  {
+    title: "Documentation",
+    link: "https://opensource.tcetmumbai.in/docs/projects/tcet-linux/about-tcet-linux",
+    newtab: true,
+  },
+  {
+    title: "About Us",
+    link: "https://opensource.tcetmumbai.in/docs/about-tcetopensource",
+    newtab: true,
+  },
 ];
 
 const Navbar = () => {
-    const [open, setOpen] = useState(false);
+  const [open, setOpen] = useState(false);
 
-    const handleMenu = () => {
-        setOpen((prev) => !prev);
-        console.log(open);
-    };
-    return (
-        <div className='bg-[#101828] font-sans'>
-            <div className="mx-auto  px-4 sm:px-6 lg:px-8">
-                <div className="flex items-center justify-between h-16">
-                    {/* Logo */}
-                    <div className="flex items-center">
-                        <a href='/' className='text-white'>
-                            <img src="tcet-opensource-logo.svg" alt="TCET Open Source Logo" srcSet="" />
-                        </a>
-                    </div>
-                    {/* Navlinks */}
-                    <div className="hidden md:flex">
-                        <div className="text-base 2xl:text-lg ml-10 flex items-baseline space-x-2">
-                            {navlinks.map((link, index) => (
-                                <a
-                                    key={index}
-                                    className="text-[#E4E7EC] transition-all duration-500 hover:bg-[#475467] hover:text-[#E0EAFF] px-2 py-1 rounded-md text-md font-medium"
-                                    href={link.link}
-                                    target=
-                                    {
-                                        link.newtab?"_blank":""
-                                    }
-                                >
-                                    {link.title}
-                                </a>
-                            ))}
+  const handleMenu = () => {
+    setOpen((prev) => !prev);
+    console.log(open);
+  };
+  return (
+    <div className="bg-[#0C1030] font-sans">
+      <div className="mx-auto  px-4 sm:px-6 lg:px-8">
+        <div className="flex items-center justify-between h-16 pl-4">
+          {/* Logo */}
+          <div
+            class="flex items-center gap-3 rounded-full "
```

Comment:
figure out gradient border

Reply:
i have added border but the background and the border keep overlapping even after adding overflow hidden attribute